kiwi/trainers/callbacks.py

```diff
@@ -88,13 +91,11 @@ def must_eval(self, epoch=None, step=None):
             return self.validation_steps and step % self.validation_steps == 0
         return False
 
-    def must_save(self, stats):
-        if self.save:
-            if self._validation_epoch <= self.keep_only_best:
-                return True
-            elif stats > self.worst_stats():
-                return True
-        return False
+    def must_save_best(self, stats):
```

Comment:
So the logic is now to always save at last the last checkpoint?

Reply:
Now we always save the latest checkpoint.

This was because when trying to keep fine-tuning a model we could never resume the run from it's last checkpoint. 

e.g. if you were doing 20 epochs and your best epoch was the 2nd, if you tried to resume training it would restart from the 2nd epoch as it was the only saved checkpoint